tedana/selection/selection_utils.py

```diff
@@ -316,50 +314,37 @@ def clean_dataframe(component_table):
 
 
 def confirm_metrics_exist(component_table, necessary_metrics, function_name=None):
-    """
-    Confirm that all metrics declared in necessary_metrics are included in comptable.
+    """Confirm that all metrics declared in necessary_metrics are already included in comptable.
 
     Parameters
     ----------
     component_table : (C x M) :obj:`pandas.DataFrame`
         Component metric table. One row for each component, with a column for
         each metric. The index should be the component number.
-    necessary_metrics : :obj:`set`
-        A set of strings of metric names
+    necessary_metrics : :obj:`list`
+        A list of strings of metric names.
     function_name : :obj:`str`
-        Text identifying the function name that called this function
-
-    Returns
-    -------
-    metrics_exist : :obj:`bool`
-        True if all metrics in necessary_metrics are in component_table
+        Text identifying the function name that called this function.
 
     Raises
     ------
     ValueError
-        If metrics_exist is False then raise an error and end the program
+        If ``metrics_exist`` is False then raise an error and end the program.
 
     Note
```

Comment:
```suggestion
    Notes
```
The underline was extended, and Notes is the appropriate section title, per the numpydoc standard.

Reply:
Most of the code uses `Notes` but I/Josh used `Note` all over `selection_nodes.py` and `selection_utils.py` and once in `ica_reclassify.py`. One of us should either fix that in this PR or in a more documentation-focused PR.

Reply:
In that case, then the added `-` in the line after this one should be removed.